packages/devextreme/js/__internal/grids/new/grid_core/columns_controller/utils.ts

```diff
@@ -26,7 +26,8 @@ function normalizeColumn(
     ?? columnFromDataOptions?.dataType
     ?? defaultColumnProperties.dataType;
   const columnDataTypeDefaultOptions = defaultColumnPropertiesByDataType[dataType];
-  const columnFormat = column.format ?? columnFromDataOptions?.format;
+  const columnFormat = column.format
+    ?? columnDataTypeDefaultOptions?.format;
```

Comment:
Let's fall back to the calculated from data format ifthe  default one isn't specified 

Or as extended solution we can specify default formats for all dataTypes and get rid of the format calculated from data (but this is optional enhancement)

```suggestion
  const columnFormat = column.format
    ?? columnDataTypeDefaultOptions?.format
    ?? columnFromDataOptions?.format;
```